lib/server/utils/ConnectionFilterUtil.js

```diff
@@ -22,15 +22,18 @@ module.exports = function(wilsonConfig) {
     return priority;
   };
 
-  var getConnectionFilterArray = function (tags) {
-    var filters = [];
+  var getConnectionFilterArray = function (tags, supportMultiValues) {
+    var filters     = [];
+    var delimiter   = '+';
```

Comment:
For now this is set to '+'.  I figure we could make this configurable, but there's not a great place for in the config structure for now, so I figured it could fixed for now.